lib/tasks/argo_testing.rake

```diff
@@ -62,7 +62,16 @@ if ['test', 'development'].include? Rails.env
 
         require 'webmock'
         # only allow connections to Fcrepo, Solr and workflow
-        WebMock.disable_net_connect!(allow: ['workflow:3000', 'solr:8983', 'fcrepo:8080', 'localhost:8983', 'localhost:8984', 'localhost:3004'])
+        WebMock.disable_net_connect!(allow: ['workflow:3000',
+                                             'solr:8983',
+                                             'fcrepo:8080',
+                                             'localhost:8983',
+                                             'localhost:8984',
+                                             'localhost:3004',
+                                             'dor-indexing-app:3000',
+                                             'dor-services-app:3000',
+                                             'techmd::3000',
```

Comment:
is this necessary for indexing?  shouldn't it only have one `:`?

Reply:
Probably not, was just adding whatever was in docker-compose to get coverage. Removed and can add later if we find it necessary.